pkg/math/oracle/median_test.go

```diff
@@ -149,7 +150,7 @@ func TestAggregateData(t *testing.T) {
 			result := m.DataAggregator.GetAggregatedData()
 			require.Equal(t, len(tc.expectedPrices), len(result))
 			for ticker, price := range result {
-				verifyPrice(t, tc.expectedPrices[ticker], price)
+				math.VerifyPrice(t, tc.expectedPrices[ticker], price, acceptableDelta)
```

Comment:
Why did these become non-deterministic now?

Reply:
they aren't non-deterministic, they just put a bound on the amount of precision we expect in the resulting calculation. This only applies for big.Ints, for big.Floats we can directly specify precision so much of this might be deprecated with the changes that are following this.